content/api/warehousing/index.html

```diff
@@ -31,18 +31,16 @@
       <link rel="stylesheet" href="warehousing.css">
       <script src="listcollapse.js" type="text/javascript" language="javascript1.2"></script>
       <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
-      <div class="flex align-ifs mbl mts">
-        <span class="mb-badge text-note mrxs pas">Base URL</span>
-        <span class="flex-auto maxw40r">
-          <pre class="flex align-ic pas mb0 wrap-text"><span>https://bapi.bring.com/wh/api/v1</span></pre>
-        </span>
-      </div>
+      <dl>
```

Comment:
Just so the spacing aligns with other pages between the heading the list:

```suggestion
      <dl class="mtm">
```